### ISSUE TITLE
every does not support thisArg

### ISSUE THREAD

**Issue by esromneb** [munrocket/overload-bracket#2]
sorry to report more issues on your first npm project but I noticed them so I will report anything I find.  Is that ok?

According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every  .every() should accept a second argument thisArg

**Comment by munrocket** [comment]
Yes of course. And if you find this project usefull you can make a PR. This will be my first review of code, you will be my first guy :D

**Comment by munrocket** [comment]
Ok, I fixed all, except this.

**Comment by esromneb** [comment]
I made a PR

**Comment by munrocket** [comment]
Wow, nvm. Thanks!
